pages/calldata-decoder.page.tsx

```diff
@@ -92,28 +93,41 @@ export default function CalldataDecoder() {
 
   return (
     <ToolLayout>
-      <h1> Calldata decoder </h1>
-
-      <label htmlFor="calldata" className="text-lg font-bold">
-        Calldata
+      <header className="mb-6 flex items-center gap-3 align-middle">
+        <Img
+          src="/static/svg/decoders.svg"
+          width={32}
+          height={32}
+          alt="deth tools logo"
+        />
+        <h3 className="text-sm text-deth-gray-300 sm:text-xl"> Decoders / </h3>
```

Comment:
I think most people just write JSX and HTML without that whitespace between tags.

```suggestion
        <h3 className="text-sm text-deth-gray-300 sm:text-xl">Decoders /</h3>
```

Take a look at HTML Paragraph docs on NPM.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p

Reply:
Sure, we can stick to that convention from now 😝